compose/agent_certs/generate-agent-certs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+set -x
+
+CERTS_DIR=$(realpath "$(dirname "$0")")
```

Comment:
This along with the `cd $CERTS_DIR` and `trap "cd -" EXIT` below are also doing the same `DIR=` trick like I described in the other comment thread, so it could be another example to follow.